src/upf_network.py

```diff
@@ -49,12 +51,70 @@ def get_ip_address(self) -> str:
             ip_addresses = iface_record.ipaddr
             for ip in ip_addresses:
                 if ip.family == AF_INET:
-                    return ip.address
+                    return f"{ip.address}/{ip.prefixlen}"
```

Comment:
Let's specify in the docstring that we are returning an IP address in this format (i.e. ex. "1.2.3.4/24")

Reply:
Unresolving as this was not done